### DISCUSSION TITLE
How to set "closest mirror" in kickstart file?

### DISCUSSION THREAD

**Discussion by bin456789** [rhinstaller/anaconda#4669] [Q&A]
I manual install an OS with choosing closest mirror in GUI.
But I can't find configs about "closest mirror" in anaconda-ks.cfg from new OS.

Later I use the same anaconda-ks.cfg to start a kictstart installation. But the installation does not skip the "installation source" part. I have to manual set the source to begin the install.

How to set "closest mirror" in kickstart file?  I don't want to set the `url --mirrorlist` string.

**Comment by VladimirSlavik** [comment]
Sorry - as far as I can tell, there is no way to do this in kickstart. I have no idea why, just that it is so.

When I test my changes, I use exactly what you mentioned (`url --mirrorlist`) to not care about source.
